src/app/(proper_react)/(redesign)/(public)/breach-details/[breachName]/BreachDetailView.tsx

```diff
@@ -128,25 +131,49 @@ export const BreachDetailsView = (props: Props) => {
         </h2>
         <div className={styles.content}>
           <p>
-            {l10n.getString("breach-overview-new", {
-              breachDate: (breach.BreachDate as unknown as Date).toLocaleString(
-                getLocale(l10n),
-                {
+            {breach.Name === "BVD" && getLocale(l10n).split("0")[0] === "en" ? (
```

Comment:
Q: What is [`getLocale()`](https://github.com/mozilla/blurts-server/blob/62162b2638fd612964572f90547a2e9751f41fe6/src/app/functions/universal/getLocale.ts) returning where we're string splitting on `0`?
The code/typing implies it returns a string, but not sure in what cases the string would have a zero as a delimiter.

Reply:
Aargh, eagle-eyed Peter strikes again! That was a typo that I didn't detect because 1) I didn't realise that we have a localisation for `en-ca` but not `en-gb` and 2) if there's no match, it'll just return the entire string. And thus I just got bucketed into the default `en`.

Fixed in 6cbaa5db9.